pkg/proto/datadog/model/v1/model.proto

```diff
@@ -73,6 +73,21 @@ message Entity {
     repeated string standardTags = 6;
 }
 
+message GenerateContainerIDFromOriginInfoRequest {
+    // Local Data
```

Comment:
Why not have 2 separate structs? One for local data and another for external?
Also, shouldn't the inode appear here for the local data?

Reply:
Done 👍 
Regarding the `inode` I was not fully certain but we can have it even if it's unused.